### ISSUE TITLE
Sampling causing errors in other packages?

### ISSUE THREAD

**Issue by IainNZ** [lindahua/Sampling.jl#1]
I'm seeing

```
ERROR: type: Sampleable: in F, expected F<:VariateForm, got Type{Univariate}
```

in many packages, which is making them unloadable.

See http://pkg.julialang.org/ -> More Options -> ... Statistics ....

**Comment by lindahua** [comment]
I think this is fixed with Distributions v0.4.10. Would you please check again to see whether things are fine now?

**Comment by IainNZ** [comment]
Fixed for quite some time
